code/game/gamemodes/events/money_lotto.dm

```diff
@@ -24,8 +24,11 @@
 /datum/event/money_lotto/start()
 	winner_sum = pick(5000, 10000, 15000)
 	if(all_money_accounts.len)
-		var/datum/money_account/D = pick(all_money_accounts)
-		winner_name = D.owner_name
+		var/list/private_accounts = all_money_accounts.Copy()
+		for(var/i in department_accounts)
```

Comment:
this is unnecessary, you can just directly subtract a whole list, like this

private_accounts -= department_accounts

Reply:
Department accounts is a list with numbers as keys and the actual accounts as value, i thought simply doing -= would attempt to subtract the keys, huh

Reply:
oh i see, disregard this comment then